docs/design/virtual-addresses.md

```diff
@@ -0,0 +1,123 @@
+# Virtual Addresses
+
+## Purpose
+Introduce the concept of virtual addresses in order to bridge the conceptual gap between Hedera's and Ethereum's view of accounts.
+Virtual addresses will resolve the issue of account EVM compatibility and identification and will be responsible for EVM address related logic.
+
+## Goals
```

Comment:
Possibly should be mentioned as an explicit goal: the seamless migration.  (Or is this just understood as a goal?  This is a design doc and migration could be a legitimate goal of its own.)

Reply:
Added seamless migration as an explicit goal.